content/en/seachat/seachat-manual/04-channels/03-seachat-meta-messenger-integration.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Facebook Messenger"
-description: "Connect SeaChat to Facebook Messenger"
+title: "Facebook Meta Messenger"
```

Comment:
original name "Facebook Messenger" is the correct name. ""Facebook Meta Messenger" sounds weird.